### PR TITLE
Update Map Template to use Mapbox v3

### PR DESCRIPTION
**Pull request by matbmapspeople**
The scope of this pull request is to make Map Template use the latest version of the Web SDK as well as use MapboxV3View.

### CODE REVIEW

File: packages/map-template/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "eslint": "^8.39.0",
     "eslint-config-react-app": "^7.0.1",
     "i18next": "^23.7.7",
-    "mapbox-gl": "^2.12.0",
+    "mapbox-gl": "3.0.0-rc.2",
```
Comment (ammapspeople) [inline packages/map-template/package.json:22]: Why aren't we using the "official" release, but a release candidate?

**Comment by andreeaceachir142** [comment]
In the `index.html` page here, should we also change the Mapbox version? @matbmapspeople 
image
cc @ammapspeople

**Comment by andreeaceachir142** [comment]
Yes, agreed 🙌🏻

**Comment by matbmapspeople** [comment]
Pull request updated! @ammapspeople @andreeaceachir142

**Review by andreeaceachir142 — APPROVED** [review]
👏🏻

File: packages/map-template/src/components/MapTemplate/MapTemplate.jsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
 
             const miSdkApiTag = document.createElement('script');
             miSdkApiTag.setAttribute('type', 'text/javascript');
-            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/4.26.4/mapsindoors-4.26.4.js.gz');
+            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/CandidateReleases/2024.22.01-rc0/mapsindoors-2024.22.1-rc0.js.gz');
```
Comment (ammapspeople) [inline packages/map-template/src/components/MapTemplate/MapTemplate.jsx:142]: This points to a candidate release. Is this really what we want to use in the Map Template?

**Review by ammapspeople — APPROVED** [review]
👏

**Pull request merged**